README.md

```diff
@@ -1,82 +1,58 @@
 # Mainframer
 
-Tool that allows you to move build process from a local machine to a remote one.
+The tool that moves build process from a local machine to a remote one.
 
-Remote machine ought to be much faster than a laptop. 
-With `mainframer` you can free up your local machine for better things —
-like editing source code in your IDE without lags and
-freezes, being able to actually _use_ your computer when the build is happening.
+Performance-focused headless machines are usually much faster than a laptop.
+Mainframer helps to move heavy operations (like compiling the source code)
+from a laptop to a dedicated machine. This way you, as a developer,
```

Comment:
I never liked that we hardcoded "laptop" here, it's not like you can't use Mainframer on desktop machines.

Maybe you have a better wording on mind?

Reply:
Changed this block to be more generic and not tied to laptops and build process.